### PR TITLE
Change MicroRPC Proto Message Names

### PR DESCRIPTION
**Pull request by pmcgrath17**
Update ```Request``` and ```Response``` proto names in micro_rpc to ```RequestWrapper``` and ```ResponseWrapper```.

#4295

### CODE REVIEW

**Comment by pmcgrath17** [comment]
Sorry, prematurely asked for review. Let me resolve all errors.

**Review by tiziano88 — APPROVED** [review]
Thanks Patrick!

File: micro_rpc/src/lib.rs [hunk 1]
```diff
@@ -102,7 +102,7 @@ impl From<Result<Vec<u8>, Status>> for proto::Response {
 /// Invokes the method identified by `method_id` via the provided [`Transport`], taking care of the
 /// serialization and deserialization over the transport.
 ///
-/// The return value has two layers of [`Result`]:
+/// The return value has two layers of [`ResultWrapper`]:
```
[the remark this answers is not in the document]
Comment (pmcgrath17) [reply micro_rpc/src/lib.rs]: I reverted this change. Thanks for the catch!

**Review by tiziano88 — APPROVED** [review]
Great, thanks

**Pull request merged**